osu.Game.Rulesets.Catch/Edit/CatchSelectionHandler.cs

```diff
@@ -52,16 +52,21 @@ public override bool HandleMovement(MoveSelectionEvent<HitObject> moveEvent)
             return true;
         }
 
-        public override bool HandleFlip(Direction direction)
+        public override bool HandleFlip(Direction direction, bool flipOverOrigin)
         {
+            if (SelectedItems.Count == 0 && !flipOverOrigin)
+                return false;
```

Comment:
What's the `flipOverOrigin` condition here about? If there are no selected items there's just never anything to do... period?

Reply:
Copy paste fail. Have removed.